packages/story-editor/src/components/checklist/checks/videoOptimization.js

```diff
@@ -50,20 +51,28 @@ const OptimizeButton = styled(Button)`
 `;
 
 export function videoElementsNotOptimized(element = {}) {
-  if (element.resource?.isTranscoding) {
+  if (!element?.resource) {
```

Comment:
`element` is never `null`, so no need for the optional chaining here.

```suggestion
  if (!element.resource) {
```